packages/utils/src/date.js

```diff
@@ -47,12 +47,26 @@ const toDayBeginInUTC = value => {
   return utcDate;
 };
 
+const getStartDay = day => {
```

Comment:
types?

Reply:
for now utils not typed

Reply:
decalred in `types/date.d.ts`

Reply:
done

Reply:
hmm, you can try to copy tsconfig.js from `class` package and fix declares